From 950b2ea99adf85ca3ebd2f251af839966c161c77 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Thu, 03 Oct 2024 12:10:23 +0300
Subject: [PATCH] [FSW-10021] fs1: fix dest sram shift mask

With this patch corrected the destination sram
type bit position with proper shift macro.

Change-Id: I77126370ff15ac12c09a0cf49fd306ddcb73bdda
---

diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index 5a27150..cd67c39 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -344,7 +344,7 @@
 	spi_dma_ctrl = PARC_GLOBAL_CONF_SPI_DMA_CTRL_EN_M;
 
 	/* sram=1 hbw=0 as destination */
-	spi_dma_ctrl |= fw_sram_dst << PARC_GLOBAL_CONF_SPI_DMA_CTRL_DST_SRAM_M;
+	spi_dma_ctrl |= fw_sram_dst << PARC_GLOBAL_CONF_SPI_DMA_CTRL_DST_SRAM_S;
 
 	/* # of 4bytes to DMA copy */
 	spi_dma_ctrl |= (uint32_t)image_size_4b << PARC_GLOBAL_CONF_SPI_DMA_CTRL_MEM_SIZE_S;
